packages/openapi-fetch/src/index.js

```diff
@@ -663,3 +670,13 @@ export function removeTrailingSlash(url) {
   }
   return url;
 }
+
+/**
+ * Validate middleware object
+ * @type {import("./index.js").validateMiddleware}
+ */
+export function validateMiddleware(middleware) {
```

Comment:
I'm not sure I'm comfortable exposing this (also, it seems unused, intended?).

My concern is that it is necessarily a heuristic (we cannot validate the types of the member functions).